doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -227,7 +227,7 @@ NCSDK-31528: Deadlock on system workqueue with ``tx_notify`` in host
   This does not guarantee that the problem is solved, as multiple blocking calls might exhaust the buffer pool.
   You can also use the (experimental) :kconfig:option:`CONFIG_BT_CONN_TX_NOTIFY_WQ` Kconfig option to use a separate workqueue for connection TX notify processing.
 
-.. rst-class:: v3-0-2 v3-0-1 v3-0-0 v2-9-0-nRF54H20-1 v2-9-2 v2-9-1 v2-9-0 v2-8-0 v2-7-0 v2-6-4 v2-6-3 v2-6-2 v2-6-1 v2-6-0
+.. rst-class:: v3-1-0 v3-0-2 v3-0-1 v3-0-0 v2-9-0-nRF54H20-1 v2-9-2 v2-9-1 v2-9-0 v2-8-0 v2-7-0 v2-6-4 v2-6-3 v2-6-2 v2-6-1 v2-6-0
 
 DRGN-24352: Missing disconnection events when HCI Controller to host flow control is enabled
```

Comment:
DRGN-24352 has been renamed to NCSDK-32999. To better track this issue, please update the ticket name